devel/ccache-devel/Portfile

```diff
@@ -25,24 +26,23 @@ long_description \
     compilation can be satisfied from cache.  This often results in a 5 to 10 \
     times speedup in common compilations.
 
-homepage            https://ccache.samba.org/
+homepage            https://ccache.dev
+
+# gcc 4.x clang < 500 does not understand -Wpedantic
+compiler.blacklist *gcc-4.* {clang < 500}
 
 # use_autoconf yes and setting autoconf.cmd to a non-default value will modify
 # depends_build, removing autoconf/automake/libtool. Avoid this by doing it
 # before setting depends_build.
 use_autoconf        yes
 autoconf.cmd        ./autogen.sh
 
-depends_build       port:asciidoc \
-                    port:autoconf \
+depends_build       port:autoconf \
                     port:automake \
                     port:libtool \
                     bin:perl:perl5
 
-depends_lib         port:zlib
-
-patchfiles          no-Werror.patch
```

Comment:
This patch is no longer necessary? The build no longer uses `-Werror`?

Reply:
Correct, builds fine without it, and the patch anyway failed to apply.

Reply:
Also note, the `cache` port is using the exact same version, and does not use any such patch.

Reply:
I didn't mean "does it build", I meant "does it use `-Werror`?" If it uses `-Werror`, we don't want it to, and we should patch it to remove it. I don't remember why I only added the patch to ccache-devel at the time; whatever is decided should be applied to both.

Reply:
To be honest because the `ccache` port was already not using the patch for the same build, and because it failed to apply, I just removed it and checked it built. I did not pay any attention to what flags may or may not have been used. I also assumed `-Werror` was only removed because it was causing build errors, so if it built fine I presumed it was no longer needed, no ?

Reply:
I just had a look, and as far as I can tell `-Werror` does not appear in the generated Makefile for the build, so is no longer used.